Lib/http/client.py

```diff
@@ -1419,8 +1419,7 @@ def __repr__(self):
             e = ''
         return '%s(%i bytes read%s)' % (self.__class__.__name__,
                                         len(self.partial), e)
-    def __str__(self):
-        return repr(self)
+    __str__ = object.__str__
```

Comment:
(just because I'm curious) in some cases this line was removed, in some cases it was moved to `object.__str__` -- is there a heuristic you used here or ?

Reply:
It is removed if no parent class except `object` defines `__str__`. But if it was changed in the parent class (in `BaseException` in this case) we need to set it explicitly.

Reply:
Awesome, thanks for the clarification 👍